debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Scn.java

```diff
@@ -27,10 +28,20 @@
 
     private final BigInteger scn;
 
+    private OffsetDateTime time;
```

Comment:
Based on the above recommendation, we would remove any changes made to the `Scn` domain class, removing the new time field and the new getting/setter methods.